src/ppx/application.cpp

```diff
@@ -668,6 +672,27 @@ void Application::DispatchShutdown()
 {
     Shutdown();
 
+    if (mSettings.useMetrics) {
+        using namespace std;
```

Comment:
note: that's forbidden by the google c++ style.

(we have been using `using namespace ppx` all across the project, I know..)

Reply:
Oh, I'll change it. :)

Reply:
Fixed.